scheduler.py

```diff
@@ -22,9 +24,22 @@ def submit_task(self, task : Task) -> Future:
         logging.info(f"Task {task} submitted to worker:{random_worker}")
```

Comment:
The flow would break here, since in workerConnectInfo: "hostname,port" is used. But here workerClient is created by splitting by ":" 
Can you please fix this?